packages/facility-server/app/routes/apiv1/suggestions.js

```diff
@@ -551,13 +551,44 @@ createSuggester(
   (search) => ({
     [Op.or]: [
       Sequelize.where(
-        Sequelize.fn('concat', Sequelize.col('first_name'), ' ', Sequelize.col('last_name')),
+        Sequelize.fn(
+          'concat',
+          Sequelize.fn('trim', Sequelize.col('first_name')),
+          ' ',
+          Sequelize.fn('trim', Sequelize.col('last_name')),
+        ),
```

Comment:
_Optional_:
We can make it a bit more reusable and probably easier to read too if we extract the logic like this:
```
const trimColumn = (column) => Sequelize.fn('trim', Sequelize.col(column));
```

Reply:
Good call, done!